src/bsdutils/meson.build

```
@@ -1,4 +1,7 @@
+dep_fts = cc.find_library('fts', required: not cc.has_function('fts_read'))
```

@eli-schwartz This doesn't use dependency() but it works.

(Suggested by Hello71 in \#musl on Libera Chat : https://sources.debian.org/src/tofi/0.8.1-1/meson.build/?hl=133#L133)

---

Indeed, I mentioned the same, too. Still:

```suggestion
dep_fts = cc.find_library('fts', required: not cc.has_function('fts_read', prefix: '#include <fts.h>'))
```

The code I suggested above is a few more lines, but it avoids running the compiler twice if the `cc.has_function()` stage says that it was found.

---

> Indeed, I mentioned the same, too. Still:

For some reason I wasn't getting email notifications. I didn't see your comments... Looks like they match up pretty well with what was suggested in \#musl I appreciate you going out of your way to help.